.github/workflows/performance.yml

```diff
@@ -0,0 +1,151 @@
+# This is a workflow triggered by PR or triggered manually
+# Runs quick performance tests and reports the comparison against HEAD
+# Test should take less than 10 minutes to run on current self-hosted devices
+name: "Performance Testing"
+
+# Controls when the action will run.
+# Workflow runs when manually triggered using the UI or API.
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+# Env variables
+env:
+  SG_COMMIT: 649509c
```

Comment:
Is it possible to use the `main` branch of sightglass? (unsure if this is largely for testing or whether it's intended that we periodically update this)

Reply:
This is a ref from the main branch of sightglas .. it's actually HEAD as of today. However I wanted to talk about versioning with sightglass so as to be more transparent about what we are using to run tests.